auth-web/src/services/von.services.ts

```diff
@@ -0,0 +1,10 @@
+import { AutoCompleteResponse } from '@/models/AutoComplete'
+import { AxiosResponse } from 'axios'
+import ConfigHelper from '@/util/config-helper'
+import { axios } from '@/util/http-util'
+
+export default class VonService {
+  public static async getAutoComplete (searchValue: string): Promise<AxiosResponse<AutoCompleteResponse>> {
+    return axios.get(`${ConfigHelper.getVonAPIUrl()}/search/autocomplete?q=${searchValue}`)
```

Comment:
check service will accept limit on get, so we can avoid slicing from frontend

Reply:
Unfortunately, there is no provision to get paginated results 
https://github.com/bcgov/TheOrgBook/blob/master/APISpec/openapi.yml